### DISCUSSION TITLE
How to take inputs?

### DISCUSSION THREAD

**Discussion by AnirudhG07** [Textualize/textual#4925] [Q&A]
Say i want to make a question air through Textual.
I want to have single choice, multi choice and integer/string inputs. LIke web, but now through textual.
How to do it? and my main question is how to return the output checked/written by the user?

Can i get some references?

**Comment by darrenburns** [comment]
I strongly encourage you to try the tutorial: https://textual.textualize.io/tutorial/

After that, you'll probably want to look at the [guide](https://textual.textualize.io/guide/) and the [widget gallery](https://textual.textualize.io/widget_gallery/) to understand the tools available to you. There are several widgets available which might fit your needs.

**Reply by AnirudhG07** [reply]
Alright thanks.
